validator/src/main.rs

```diff
@@ -2259,7 +2286,7 @@ pub fn main() {
             .ok()
             .or_else(|| Some(ledger_path.clone())),
         dev_halt_at_slot: value_t!(matches, "dev_halt_at_slot", Slot).ok(),
-        cuda: matches.is_present("cuda"),
+        cuda,
```

Comment:
Can `ValidatorConfig::cuda` be completely removed now?

Reply:
Looks like it, yeah. I'll toss that in as well